RevenueCat/Plugins/iOS/RCPurchases+HybridAdditions.h

```diff
@@ -0,0 +1,19 @@
+//
+// Created by RevenueCat on 3/19/20.
+// Copyright (c) 2020 Purchases. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "RCPurchases.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+
+@interface RCPurchases (HybridAdditions)
+
+- (void)_setPushTokenString:(nullable NSString *)pushToken;
```

Comment:
should this be a part of common, since it's used by more than one hybrid sdk?

Reply:
This is part of common actually. We don't have the `.a` with all these files in Unity yet

Reply:
my bad, I was looking for the word "common" in the path, I forgot they're just loose files in Unity.